contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
Does this mean that you want _NamedPipeTest_ and _WinconTest_ to be moved there as well ?
(and I don't take it personally... :-))